Sources/Orbit/Support/Layout/HorizontalScrollReader.swift

```diff
@@ -0,0 +1,110 @@
+import SwiftUI
+
+@available(iOS 14, *)
+public class HorizontalScrollViewProxy: ObservableObject {
+
+    @Published fileprivate var scrollTarget: ScrollTargetValue?
+
+    public func scrollTo<ID>(_ id: ID, animated: Bool = true) where ID : Hashable {
```

Comment:
Ok, though `some Hashable` can still be done. Maybe even on the other method, since `AnyHashable` is also `Hashable`...

Reply:
Updated this one to use `some Hashable`, but kept it in the method inside the other component - otherwise `as AnyHashable` would need to be cast in the `first()` method.